tinygrad/runtime/ops_llvm.py

```diff
@@ -12,13 +12,15 @@ def expect(x, err, ret=None):
   return ret
 
 class LLVMCompiler(Compiler):
-  def __init__(self, host_arch:str):
-    for component in ['Target', 'TargetInfo', 'TargetMC', 'AsmParser', 'AsmPrinter']: getattr(llvm, f'LLVMInitialize{host_arch}{component}')()
+  def __init__(self, target_arch:str, gpu:str|None=None):
+    for component in ['Target', 'TargetInfo', 'TargetMC', 'AsmParser', 'AsmPrinter']: getattr(llvm, f'LLVMInitialize{target_arch}{component}')()
 
-    triple = {'AArch64': b'aarch64', 'X86': b'x86_64'}[host_arch] + b'-none-unknown-elf'
+    if target_arch == "AMDGPU": triple, cpu, feats = b"amdgcn-amd-amdhsa", (gpu or "gfx1100").encode(), b"+cumode"
```

Comment:
So this isn't how this works. This isn't a standalone improvement, this is just a random piece of the other PR